doc-content/enterprise-only/admin-and-config/managing-business-central-using-usergroupcallback-ref.adoc

```diff
@@ -0,0 +1,55 @@
+[id='managing-business-central-using-usergroupcallback-ref']
+= Using LDAP UserGroupCallback implementation
+
+You can use the LDAP `UserGroupCallback` implementation by configuring the respective LDAP properties in one of the following ways:
+
+* *_programatically_*: build a _Properties_ object with the respective `LDAPUserGroupCallbackImpl` properties and create `LDAPUserGroupCallbackImpl` with the _Properties_ object as its parameter.
++
+[source]
+----
+import org.kie.api.PropertiesConfiguration;
+import org.kie.api.task.UserGroupCallback;
+...
+Properties properties = new Properties();
+properties.setProperty(LDAPUserGroupCallbackImpl.USER_CTX, "ou=People,dc=my-domain,dc=com");
+properties.setProperty(LDAPUserGroupCallbackImpl.ROLE_CTX, "ou=Roles,dc=my-domain,dc=com");
+properties.setProperty(LDAPUserGroupCallbackImpl.USER_ROLES_CTX, "ou=Roles,dc=my-domain,dc=com");
+properties.setProperty(LDAPUserGroupCallbackImpl.USER_FILTER, "(uid={0})");
+properties.setProperty(LDAPUserGroupCallbackImpl.ROLE_FILTER, "(cn={0})");
+properties.setProperty(LDAPUserGroupCallbackImpl.USER_ROLES_FILTER, "(member={0})");
+
+UserGroupCallback ldapUserGroupCallback = new LDAPUserGroupCallbackImpl(properties);
+
+UserGroupCallbackManager.getInstance().setCallback(ldapUserGroupCallback);
+----
+
+* *_declaratively_*: create the `jbpm.usergroup.callback.properties` file in the root of your application or specify the file location as a system property: `-Djbpm.usergroup.callback.properties=FILE_LOCATION_ON_CLASSPATH`
++
+Ensure that you register the LDAP callback when starting the _User Task_ server.
++
+[source]
+----
+#ldap.bind.user=
+#ldap.bind.pwd=
+ldap.user.ctx=ou\=People,dc\=my-domain,dc\=com
+ldap.role.ctx=ou\=Roles,dc\=my-domain,dc\=com
+ldap.user.roles.ctx=ou\=Roles,dc\=my-domain,dc\=com
+ldap.user.filter=(uid\={0})
+ldap.role.filter=(cn\={0})
+ldap.user.roles.filter=(member\={0})
+#ldap.user.attr.id=
+#ldap.roles.attr.id=
+----
+
+[float]
+=== Additional resources
+
+* For information on roles and users, see https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.1/html-single/installing_and_configuring_red_hat_process_automation_manager_on_red_hat_jboss_eap_7.1/roles-users-con/[_Roles and users_].
```

Comment:
* For information on roles and users, see 
Delete these intros and search for similar in the docs

Reply:
Changed as suggested.